error: variable 'supp_size' set but not used

```
hb-subset-cff1.cc:405:33: error: variable 'supp_size' set but not used [-Werror,-Wunused-but-set-variable]
    unsigned int  size0, size1, supp_size;
                                ^
1 error generated.
make[4]: *** [libharfbuzz_subset_la-hb-subset-cff1.lo] Error 1
make[4]: *** Waiting for unfinished jobs....
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

```

CXX      libharfbuzz_subset_la-hb-subset-cff2.lo
  CXX      libharfbuzz_subset_la-hb-subset-input.lo
hb-subset-cff1.cc:405:33: error: variable 'supp_size' set but not used [-Werror,-Wunused-but-set-variable]
    unsigned int  size0, size1, supp_size;
                                ^
  CXX      libharfbuzz_subset_la-hb-subset-plan.lo
1 error generated.
make[4]: *** [libharfbuzz_subset_la-hb-subset-cff1.lo] Error 1
make[4]: *** Waiting for unfinished jobs....
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1

--- a/build.bash
+++ b/build.bash
@@ -342,6 +342,9 @@ function build_harfbuzz () {
     echo '♻️ ' Start compiling harfbuzz
     cd ${CMPLD}
     cd harfbuzz-2.7.2
+    sed -i -e 's/unsigned int  size0, size1, supp_size;/unsigned int  size0, size1;/g' ./src/hb-subset-cff1.cc
+    sed -i -e 's/supp_size = 0;//g' ./src/hb-subset-cff1.cc
+    sed -i -e 's/supp_size += SuppEncoding::static_size \* supp_codes.length;//g' ./src/hb-subset-cff1.cc
     ./configure --prefix=${SRC} --disable-shared --enable-static
     make -j ${NUM_PARALLEL_BUILDS}
     make install

